packages/core/src/models/Slice.ts

```diff
@@ -32,11 +32,11 @@ export type VariationSM = t.TypeOf<typeof VariationSM>;
 
 export const VariationMock = t.type({
   variation: t.string,
-  name: t.string,
```

Comment:
do we not need a name at all then?

Reply:
It's also been removed in the prismic-mocks package